### PR TITLE
responsively hiding columns

### PR DESCRIPTION
**Pull request by csabapalfi**
Although the `hidden-*` classes are not just for columns they are really frequently used on them.

### CODE REVIEW

**Comment by taion** [comment]
Can you add a quick test for this?

**Comment by AlexKVal** [comment]
It seems it would be nice to have commits "squashed".

Also the header of the resulted commit should begin with `[added] ...` tag. [Here is more about it.](https://github.com/react-bootstrap/react-bootstrap/blob/master/CONTRIBUTING.md#commit-subjects-for-public-api-changes)
That way this change will be reflected in the [Changelog](https://github.com/react-bootstrap/react-bootstrap/blob/master/CHANGELOG.md)

:cherries:

**Comment by csabapalfi** [comment]
Done. Thanks for reviewing!

Should've started by reading CONTRIBUTING.md. :blush:

**Comment by AlexKVal** [comment]
LGTM :sparkle: 
Though I believe it woud be nice to have an additional review by @taion or @jquense to get merged.

File: src/Col.js [hunk 1]
```diff
@@ -38,6 +38,38 @@ const Col = React.createClass({
      */
     lg: React.PropTypes.number,
     /**
+     * Hide column
+     *
+     * on Extra small devices Phones
+     *
+     * adds class `col-xs-hidden`
```
Comment (jquense) [inline src/Col.js]: This isn't actually correct, right? the classes are just [size]-hidden

Comment (csabapalfi) [inline src/Col.js]: Ouch, fixing comment now.

**Comment by csabapalfi** [comment]
Now I also fixed propTypes comments. Thanks @jquense!

**Comment by taion** [comment]
:shipit:

**Comment by AlexKVal** [comment]
@csabapalfi Thank you for the contribution :wink: :cherries:

**Pull request merged**